docs/src/dMCDA.md

```diff
@@ -1 +1,159 @@
+|```coral_cover_tol```        |% of area of corals to be seeded allowed on a site when selecting sites for seeding|
+|```deployed_coral_risk_tol```|Risk tolerance for heat and wave risk when seeding or shading corals.              |
+|```depth_min```              |Minimum depth of sites when selecting for seeding or shading corals.               |
+|```depth_offset```           |depth_min+depth_offset is maximum depth of sites when seeding or shading corals.   |
+
+## MCDA methods
+Multi-criteria decision analysis encompasses a series of methods for conducting decision making in a way that is formalised, structured and transparent. It evaluates the decision objective according to numerical measures of decision criteria assembled by the decision maker. Constructing explicit criteria over which to evaluate alternatives allows a transperant evaluation of benefits, negatives and trade-offs in coming to a decision solution.
+
+Commonly, comparison of criteria in MCDA is acheived through the construction of a “values matrix” or “decision matrix”, which contains measures of value for each alternative against an array of criteria. The measure of value, for example, could be a relative ranking of worth, probability of success, or risk of failure. Formally, these values form an $N$ by $M$ matrix where entry $x_{i,j}$ is the value of the $i$th alternative as evaluated against the $j$th criterium. The decision matrix is generally normalised, so that
+
+$$\sum_{i=1}^N\sqrt{x_{i,j}^2}=1 \forall j.$$
+
+The user also weights each criterium with a weighting representing it's relative importance in the decison problem (which could, for example, be decided through stakeholder engagement or expert opinion). These weightings $w_j, j =1,...,M$, are also normalised so that
+$$\sum_{j=1}^M w_j=1.$$
+
+The final decision matrix used in the criteria ranking algorithms in ADRIA has the form,
+$$X_{i,j} = w_j x_{i,j}$$
+
+The particular MCDA technique chosen prescribes a strategy for aggregating criteria in the decision matrix and forming rankings of alternatives. ADRIA allows the user to select from 3 main aggregation methods, which are set through the ```guided``` parameter:
+
+1. Order-ranking : ```guided = 1```
+2. TOPSIS : ```guided = 2```
+3. VIKOR : ```guided = 3```
+
+The ```guided``` parameter can be set in the input parameter DataFrame or inline, as in the example below.
+
+```julia
+@info "Loading data package"
+here = @__DIR__
+dom = ADRIA.load_domain(joinpath(here, "Example_domain"))
+
+@info "Creating 5 scenarios based on parameter bounds using the Sobol' method"
+scens = ADRIA.sample(dom, 5)
+
+@info "Setting the guided parameter for each run"
+scens.guided[1] = -1.0 # counterfactual run
+scens.guided[2] = 0.0 # unguided selection (randomised)
+scens.guided[3] = 1.0 # order ranking
+scens.guided[4] = 2.0 # TOPSIS
+scens.guided[5] = 3.0 # VIKOR
```

Comment:
There is a "usage" sub-section in the docs - I suggest we move examples of use there.